scripts/gulp/frontend-shared.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const cp = require('child_process');
+const fs = require('fs');
+const gulp = require('gulp');
+const babel = require('gulp-babel');
+const sourcemaps = require('gulp-sourcemaps');
+
+const buildFrontendSharedJs = () => {
+  // There does not appear to be a simple way of forcing gulp-babel to use a config
+  // file. Load it up as JSON and pass it in manually.
+  const babelConfig = JSON.parse(
+    fs.readFileSync('./frontend-shared/babel.config.json')
```

Comment:
This ensures we use the same babel config for npm publish as well as building locally.